cluster/saltbase/salt/fluentd-es/fluentd-es.yaml

```diff
@@ -12,6 +12,10 @@ spec:
     resources:
       limits:
         cpu: 100m
+        memory: 200Mi
+      requests:
+        cpu: 100m
+        memory: 200Mi
```

Comment:
Yeah, fluentd-gcp has been running with 200Mi since last July, and Dawn just missed this file when she added the limit there.

Our buffer_chunk_limit and buffer_queue_limit settings in the fluentd config keep it from going over this.